cupy/cuda/thrust.pyx

```diff
@@ -14,6 +14,9 @@ from cupy.cuda cimport stream
 from cupy_backends.cuda.api cimport runtime
 
 
+available = True
```

Comment:
Could you move this line to around line 69 (in the Python Interface section)?

Reply:
Fixed!